src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/integration/health/HealthCheckTest.kt

```diff
@@ -109,7 +110,33 @@ class HealthCheckTest : IntegrationTestBase() {
       get("/health/ping").willReturn(
         aResponse()
           .withHeader("Content-Type", "application/json")
-          .withBody(if (status == 200) "pong" else "some error")
```

Comment:
I don't know why this worked.  The deployed end-points are definitely returning JSON like that below.